viewer/packages/camera-manager/src/StationaryCameraManager.ts

```diff
@@ -157,15 +164,33 @@ export class StationaryCameraManager implements CameraManager {
     this._stopEventTrigger.dispose();
   }
 
-  private readonly enableDragging = (_: PointerEvent) => {
+  private enableDragging(_: PointerEvent) {
     this._isDragging = true;
-  };
+  }
 
-  private readonly disableDragging = (_: PointerEvent) => {
+  private disableDragging(_: PointerEvent) {
     this._isDragging = false;
+  }
+
+  private readonly onPointerUp = (event: PointerEvent) => {
+    const eventIndex = this._downEventCache.findIndex(downEvent => event.pointerId === downEvent.pointerId);
+    if (eventIndex > -1) {
+      this._downEventCache.splice(eventIndex, 1);
+    }
```

Comment:
I believe lodash has a utility method for this. See: https://lodash.com/docs/4.17.15#pull

Reply:
Note that this will be irrelevant if you change to a map which is suggested in a comment below.

Reply:
Replaced this with _.remove 👍